modmesh/gui/sample_mesh.py

```diff
@@ -0,0 +1,204 @@
+        (27, 2), (28, 2), (29, 2), (0, 3), (1, 3), (2, 3), (3, 3), (4, 3),
+        (5, 3), (6, 3), (7, 3), (8, 3), (9, 3), (11, 3), (12, 3), (13, 3),
+        (15, 3), (16, 3), (17, 3), (18, 3), (19, 3), (20, 3), (21, 3),
+        (22, 3), (23, 3), (24, 3), (25, 3), (26, 3), (27, 3), (28, 3),
+        (29, 3), (0, 4), (1, 4), (2, 4), (3, 4), (4, 4), (5, 4), (6, 4),
+        (7, 4), (8, 4), (9, 4), (11, 4), (12, 4), (13, 4), (15, 4),
+        (16, 4), (17, 4), (18, 4), (19, 4), (20, 4), (21, 4), (22, 4),
+        (23, 4), (24, 4), (25, 4), (26, 4), (27, 4), (0, 5), (1, 5),
+        (2, 5), (3, 5)
+    ]
+    mh.cltpn.ndarray[:] = [
+        Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,
+        Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 0-20
+        Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 21-31
+        Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 32-45
+        Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 46-61
+        Q, Q, Q  # 62-64
+    ]
+    mh.clnds.ndarray[:, :5] = [
+        (4, 0, 1, 30, 29),
+        (4, 1, 2, 31, 30),
+        (4, 2, 3, 32, 31),
+        (4, 4, 5, 34, 33),
+        (4, 5, 6, 35, 34),
+        (4, 6, 7, 36, 35),
+        (4, 8, 9, 38, 37),
+        (4, 9, 10, 39, 38),
+        (4, 10, 11, 40, 39),
+        (4, 12, 13, 42, 41),
+        (4, 13, 14, 43, 42),
+        (4, 14, 15, 44, 43),
+        (4, 15, 16, 45, 44),
+        (4, 17, 18, 47, 46),
+        (4, 18, 19, 48, 47),
+        (4, 19, 20, 49, 48),
+        (4, 21, 22, 51, 50),
+        (4, 22, 23, 52, 51),
+        (4, 23, 24, 53, 52),
+        (4, 25, 26, 55, 54),
+        (4, 27, 28, 57, 56),
+        (4, 31, 32, 61, 60),
+        (4, 33, 34, 63, 62),
+        (4, 35, 36, 65, 64),
+        (4, 37, 38, 67, 66),
+        (4, 41, 42, 69, 68),
+        (4, 44, 45, 71, 70),
+        (4, 46, 47, 73, 72),
+        (4, 50, 51, 75, 74),
+        (4, 52, 53, 77, 76),
+        (4, 54, 55, 79, 78),
+        (4, 56, 57, 81, 80),
+        (4, 58, 59, 83, 82),
+        (4, 59, 60, 84, 83),
+        (4, 60, 61, 85, 84),
+        (4, 62, 63, 87, 86),
+        (4, 64, 65, 89, 88),
+        (4, 66, 67, 91, 90),
+        (4, 68, 69, 94, 93),
+        (4, 70, 71, 96, 95),
+        (4, 72, 73, 99, 98),
+        (4, 74, 75, 103, 102),
+        (4, 76, 77, 105, 104),
+        (4, 78, 79, 107, 106),
+        (4, 79, 80, 108, 107),
+        (4, 80, 81, 109, 108),
+        (4, 82, 83, 111, 110),
+        (4, 86, 87, 115, 114),
+        (4, 87, 88, 116, 115),
+        (4, 88, 89, 117, 116),
+        (4, 90, 91, 119, 118),
+        (4, 92, 93, 121, 120),
+        (4, 93, 94, 122, 121),
+        (4, 95, 96, 124, 123),
+        (4, 96, 97, 125, 124),
+        (4, 98, 99, 127, 126),
+        (4, 99, 100, 128, 127),
+        (4, 100, 101, 129, 128),
+        (4, 102, 103, 131, 130),
+        (4, 103, 104, 132, 131),
+        (4, 104, 105, 133, 132),
+        (4, 106, 107, 135, 134),
+        (4, 110, 111, 137, 136),
+        (4, 111, 112, 138, 137),
+        (4, 112, 113, 139, 138)
+    ]
```

Comment:
How do you generate the testcase?
The example looks excessive long.

Reply:
I didn't.  Kudos to @Yu-Rou-Weng who did it by hands.